config/locales/es.yml

```diff
@@ -1658,10 +1658,10 @@ es:
           title: "¡Parece que ya ha comenzado!"
         already_filed:
           help_text_html: |
-            <p>Presentó una declaración de impuestos este año, después del 1o de enero de %{calendar_year}, para el año fiscal 2020?</p>
-            <p>Conteste que sí, incluso si todavía está esperando los pagos del estímulo, o si utilizó la Herramienta de inscripción para No Declarantes. Si lo hizo, le brindaremos más orientación sobre sus opciones en la página siguiente.</p>
-            <p class="text--bold">Si ya presentó una declaración de impuestos de %{current_tax_year}, el IRS rechazará la declaración que presente con esta herramienta.</p>
-          title: "¿Presentó su declaración de impuestos de %{current_tax_year} este año?"
+            <p>¿Presentó la declaración de impuestos este año, después del 1 de enero de %{calendar_year}, para el año fiscal %{current_tax_year}?</p>
+            <p>Responda que sí, incluso si todavía está esperando los pagos del estímulo, o si utilizó la Herramienta de inscripción para no declarantes. Si lo hizo, lo ofreceremos más orientación sobre sus opciones en la siguiente página.</p>
+            <p class="text--bold">Si ya presentó una declaración de impuestos para%{current_tax_year} la declaración que presente con esta herramienta será rechazada por el IRS.</p>
```

Comment:
This line `para%{current_tax_year}` introduces a problem, but I left a comment in Slack, so I feel good that this may get resolved, so I'll roll forward.